shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/app/ReplicationRunner.java

```diff
@@ -32,14 +32,26 @@
 import com.hotels.shunting.yard.common.event.SerializableInsertEvent;
 import com.hotels.shunting.yard.common.event.SerializableListenerEvent;
 import com.hotels.shunting.yard.common.messaging.MessageReader;
+import com.hotels.shunting.yard.common.metrics.MetricsConstant;
+import com.hotels.shunting.yard.common.metrics.MetricsHelper;
 import com.hotels.shunting.yard.common.receiver.ShuntingYardMetaStoreEventListener;
 
 @Component
 class ReplicationRunner implements ApplicationRunner, ExitCodeGenerator {
-  private static final Logger LOG = LoggerFactory.getLogger(ReplicationRunner.class);
+  private static final Logger log = LoggerFactory.getLogger(ReplicationRunner.class);
 
-  private ShuntingYardMetaStoreEventListener listener;
-  private MessageReader messageReader;
+  static void success() {
```

Comment:
Do these two need to be static?

Reply:
they don't depend on the class itself

Reply:
Then perhaps they shouldn't be here ;) Alternatively they could be private and non-static, what's the point of making something static if its never going to be called statically?

Reply:
you need to look at how the bytecode is generated

Reply:
We should never need to look at the byte code unless there's some huge performance issue that requires it.

Reply:
Since these aren't being used in a static manner let's make them private and non-static for now.

Reply:
they are used in a static manner, I can move them to an utility class if this is an issue

Reply:
I only see them being used on lines 96 and 98 and these both have access to the methods non-statically don't they? Are the methods being called from elsewhere?